.gitignore

```diff
@@ -73,6 +73,7 @@ public/ui/
 
 # spec/
 spec/replication/util/data/euwe_migrations
+spec/replication/util/data/previous_migrations
```

Comment:
master has *_migrations...can this PR just do the same to keep it consistent?

https://github.com/ManageIQ/manageiq/blob/master/.gitignore#L76-L77